blog/2023-08-12-mlops-more-oops-than-ops/index.md

```diff
@@ -480,45 +457,38 @@ class TrtDiffusionModel:
             padding="max_length",
             max_length=max_seq_length,
             truncation=True,
-            return_tensors="pt",
-        )
+            return_tensors="pt")
         text_embeddings = self.text_encoder(text_input.input_ids.to(self.device))[0]
         uncond_input = self.tokenizer(
             [""] * batch_size,
             padding="max_length",
             max_length=max_seq_length,
-            return_tensors="pt",
-        )
+            return_tensors="pt")
         uncond_embeddings = self.text_encoder(uncond_input.input_ids.to(self.device))[0]
         text_embeddings = torch.cat([uncond_embeddings, text_embeddings])
 
         latents = torch.randn((batch_size, 4, height // 8, width // 8)).to(self.device)
         self.scheduler.set_timesteps(num_inference_steps)
-
         latents = latents * self.scheduler.sigmas[0]
+
         with torch.inference_mode(), autocast("cuda"):
-            for i, t in tqdm(enumerate(self.scheduler.timesteps)):
+            for i, t in tenumerate(self.scheduler.timesteps):
```

Comment:
oh wow didn't know about `tenumerate`

Reply:
mentioned in https://github.com/tqdm/tqdm#faq-and-known-issues :)